src/set_theory/game/basic.lean

```diff
@@ -554,12 +575,66 @@ def inv' : pgame → pgame
   ⟨inv_ty l' r ff, inv_ty l' r tt,
     inv_val L' R IHl' IHr, inv_val L' R IHl' IHr⟩
 
-/-- The inverse of a surreal number in terms of the inverse on positive surreals. -/
+theorem inv'_pos : ∀ {x : pgame}, 0 < inv' x
+| ⟨xl, xr, xL, xR⟩ := by { convert lt_mk inv_ty.zero, refl }
+
+def inv'_zero : relabelling (inv' 0) 1 :=
+begin
+  change relabelling (inv' (mk _ _ _ _)) 1,
+  rw inv',
+  refine ⟨_, _, λ i, _, is_empty_elim⟩,
+  { dsimp, apply equiv_punit_of_unique },
+  { dsimp, apply equiv.equiv_pempty },
+  { simp }
+end
+
+theorem inv'_zero_equiv : inv' 0 ≈ 1 := inv'_zero.equiv
+
+def inv'_one : relabelling (inv' 1) 1 :=
+begin
+  change relabelling (inv' (mk _ _ _ _)) 1,
+  rw inv',
+  haveI H : is_empty {i : punit // (0 : pgame) < 0} := ⟨λ ⟨_, h⟩, pgame.lt_irrefl 0 h⟩,
+  refine ⟨_, _, λ i, _, is_empty_elim⟩,
+  { simp, apply @equiv_punit_of_unique _ (pgame.unique_inv_ty _ _),
```

Comment:
No idea why the instance wasn't inferred automatically.

Reply:
It's because typeclass search doesn't know that `0 < 0` is `false`.

A way to help it out:
```lean
def inv'_one : relabelling (inv' 1) (1 : pgame.{u}) :=
begin
  change relabelling (inv' (mk _ _ _ _)) 1,
  rw inv',
  generalize h : (0 : pgame.{u}) < 0 = p,
  rw lt_self_iff_false at h,
  cases h,
  refine ⟨_, _, λ i, _, is_empty_elim⟩;
  simp,
  { apply equiv_punit_of_unique },
  { apply equiv.equiv_pempty },
end
```

Reply:
I'm not quite sure what I did since posting this comment, but it seems like this works as it's currently written?
```lean
def inv'_one : relabelling (inv' 1) (1 : pgame.{u}) :=
begin
  change relabelling (mk _ _ _ _) 1,
  haveI : is_empty {i : punit.{u+1} // (0 : pgame.{u}) < 0},
  { rw lt_self_iff_false, apply_instance },
  refine ⟨_, _, λ i, _, is_empty_elim⟩;
  simp,
  { apply equiv_punit_of_unique },
  { apply equiv.equiv_pempty }
end
```
I should have resolved this comment earlier.

Reply:
I thought your question might have been "why isn't this `is_empty` instance inferred automatically?" but I see now it probably wasn't.